src/librustc_typeck/check/coercion.rs

```diff
@@ -161,6 +162,52 @@ impl<'f, 'gcx, 'tcx> Coerce<'f, 'gcx, 'tcx> {
         })
     }
 
+    fn reborrow(&self, source: Ty<'tcx>, target: Ty<'tcx>)
```

Comment:
I'm assuming this code is the same as the code you removed below, but just extracted to a function?

Reply:
Correct.